include/deal.II/fe/fe_values.h

```diff
@@ -3877,6 +3895,17 @@ class FESubfaceValues : public FEFaceValuesBase<dim, spacedim>
          const unsigned int                                     face_no,
          const unsigned int                                     subface_no);
 
+  /**
+   * Alternative reinitialization function that takes, as arguments, iterators
+   * to the face and subface instead of their numbers.
```

Comment:
It might be useful to point out that `subface` must be one of the `face->child(...)` iterators and that it must be the face iterator for a neighboring cell.